apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/edit.js

```diff
@@ -23,8 +24,29 @@ import { addQueryArgs } from '@wordpress/url';
  */
 import './style.scss';
 
+// Hide the Block Sidebar when the Template block is selected
+domReady( () => {
+	const debouncedFunction = debounce( () => {
+		const templateBlock = document.querySelector( '.template__block-container' );
+		const blockSidebarButton = document.querySelector(
+			'.edit-post-sidebar__panel-tabs ul li:last-child'
```

Comment:
do we know that it'll always be the last child?

Reply:
@noahtallen Can't say for the future, but right now it's an hardcoded list of two elements (Document and Block):

https://github.com/WordPress/gutenberg/blob/d83c23932f4ca9530ccb2c86057b3d0f52bc351c/packages/edit-post/src/components/sidebar/settings-header/index.js#L32-L53

Reply:
awesome!